admin-ui/app/routes/Apps/Gluu/GluuRemovableInputRow.tsx

```
@@ -93,11 +96,24 @@ function GluuRemovableInputRow<TValues extends FormikValues = FormikValues>({
+            width: 32,
+            height: 32,
+            padding: 6,
+          }}
+          onKeyDown={(e) => {
+            if (e.key === 'Enter' || e.key === ' ') {
+              if (e.key === ' ') e.preventDefault()
+              handler()
+            }
+          }}
           onClick={() => handler()}
         >
```

Biome (2.4.6)

[error] The HTML element with the interactive role "button" is not focusable.

(lint/a11y/useFocusableInteractive)